src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services_prototype.yml

```diff
@@ -0,0 +1,3 @@
+services:
+    Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\:
+        resources: ../Prototype
```

Comment:
I would have named it `resource`. A plural seems to indicate that the value is an array, which is not supported here.